packages/ceramic-cli/src/index.ts

```diff
@@ -11,9 +11,10 @@ program
     .option('--ethereum-rpc <url>', 'The Ethereum RPC URL used for communicating with Ethereum blockchain')
     .option('--anchor-service-api <url>', 'The anchor service URL to use')
     .option('--validate-docs', 'Validate documents according to their schemas. It is enabled by default')
+    .option('--pinning <urls...>', 'Pinning endpoints')
```

Comment:
Should it be "urls" or "url"? When you demoed you passed `--pinning` option multiple times. Maybe add that you can pass it multiple time to the option description?

Reply:
Ellipsis implies an option could be repeated multiple times, it is a usual thing. It is even mentioned in [POSIX Standard](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html). "urls" vs "url": true, singular is more conventional.

Reply:
Cool, didn't know about that 👍